api-gateway/caching/docker.gradle.kts

```diff
@@ -0,0 +1,125 @@
+/*
```

Comment:
Why do we need to copy this file?

Reply:
I've overriden some configs that allow dockerized Knot.x to reach localhost's wiremock server.